crud/select.lua

```diff
@@ -222,31 +198,32 @@ function select_module.pairs(space_name, user_conditions, opts)
         error(string.format("Failed to generate iterator: %s", err))
     end
 
-    local gen = function(_, iter)
-        if not iter:has_next() then
+    local tuple, _
+
+    if opts.use_tomap ~= true then
+        return iter.iter:pairs()
+    end
+
+    local merger_gen = iter.iter:pairs()
```

Comment:
I would like to rename `iter` field to `tarantool_iter`?
`plan.tarantool_iter`, `iter.tarantool_iter:pairs()`

Reply:
It looks like `Iterator` became `Merger`, do you think so?